Sources/CacheAdvance/EncodableMessage.swift

```diff
@@ -42,7 +42,7 @@ struct EncodableMessage<T: Codable> {
         let messageData = try encoder.encode(message)
         guard messageData.count < MessageSpan.max else {
             // We can't encode the length this message in a MessageSpan.
-            throw CacheAdvanceError.messageDataTooLarge
+            throw CacheAdvanceError.messageLargerThanCacheCapacity
```

Comment:
So `MessageSpan.max` Is the max value that can be stored in a `UInt32` IIUC.
I guess we could only hit this on a 64-bit device?

Reply:
Correct